### ISSUE TITLE
Get Instagram Page ID Fails for me.

### ISSUE THREAD

**Issue by andresgutgon** [TiagoGrosso/instagram-graph-api-lib#68]
Hi, I'm trying to get the Instagram Page ID as [described here](https://github.com/TiagoGrosso/instagram-graph-api-lib#page-id). Looking at the code and in my console what's doing this code is doing this request to Facebook

```
https://graph.facebook.com/me?access_token=ACCESS_TOKEN&l&fields=instagram_business_account'
```
But this is returning this error:
```
'OAuth "Facebook Platform" "invalid_request" "(#100) Tried accessing nonexisting field (instagram_business_account) on node type (User)
```

Reading [Facebook docs](https://developers.facebook.com/docs/instagram-api/getting-started#4--get-the-user-s-pages) it says the endpoint is this one:
```
https://graph.facebook.com/v11.0/me/accounts?access_token={access-token}
```

Which gives you all connected Facebook pages that the user has:
```
{
  "data": [
    {
      "access_token": "....",
      "category": "App Page",
      "name": "My Facebook page",
      "id": "THIS_IS_THE_FB_PAGE_ID",  // capture the Page ID
      "tasks": [...]
    }
  ]
}
```

From that request you get for example the first page `id` (`THIS_IS_THE_FB_PAGE_ID`)

And with that the [doc says](https://developers.facebook.com/docs/instagram-api/getting-started#5--get-the-page-s-instagram-business-account) you can get the Instagram ID

```
https://graph.facebook.com/v11.0/THIS_IS_THE_FB_PAGE_ID?fields=instagram_business_account&access_token={access-token}
```

And it returns the ID page ID
```
{
  "instagram_business_account": {
    "id": "THIS_IS_THE_INSTAGRAM_PAGE_ID"  // Connected IG User ID
  },
  "id": "THIS_IS_THE_FB_PAGE_ID"  // Facebook Page ID
```

So all of this looks super complicated but I think makes sense because Facebook requires 2 things:
1. To have a business IG Account. Is the only kind of accounts they allow to be managed by API
2. This IG Business Account have to be linked to a Facebook Page

I'm not sure if this changed after you built this lib or I'm totally wrong and is easier than I think : )

**Comment by TiagoGrosso** [comment]
Hmm I think this has indeed changed since I implemented it because at the time I tested it and it worked and now I'm getting the error you are describing. I'll flag this as a bug and try to work on this in the weekend. I suspect this will be a simple change

**Comment by andresgutgon** [comment]
I can't promise but I would like to work on fixing this on Sunday. Do you have doc about how to develop the lib? I mean pointing in my project to my local fork of this lib. I did once with something like linking but don't remember

**Comment by TiagoGrosso** [comment]
You can use this: https://docs.npmjs.com/cli/v7/commands/npm-link

Or you can `npm build` your local fork/clone of this lib, then `npm pack` and on your other project `npm install [link_to_tar.gz]`

**Comment by andresgutgon** [comment]
Hey @TiagoGrosso I started a PR for this issue https://github.com/TiagoGrosso/instagram-graph-api-lib/pull/69 is not finished yet but is working. Let me know what do you think : )

**Comment by TiagoGrosso** [comment]
Closed as #69 as been landed
